root/work/edit_form.tt

```diff
@@ -99,6 +99,18 @@
 
   <div class="documentation">
     [%- iswc_bubble(link_entity(work)) -%]
+
+    <div class="bubble" id="type-bubble">
+      <span id="type-bubble-default">
+        [%- l('Select any type from the list to see its description. If the work
+               doesn\'t seem to match any type, just leave this blank.') -%]
+      </span>
+      [% FOREACH id IN form.field('type_id').options %]
+         <span class="type-bubble-description" id="type-bubble-description-[%- id.value -%]">
+           <strong>Description: </strong>[%- c.model('WorkType').get_description(id.value) -%]
```

Comment:
I am not sure the label `Description:` is really necessary. If it is, it should use the macro `add_colon` for i18n.

Reply:
Personally, I think it makes it clear for someone who didn't fully read the initial message what the bubble is.